cygwin.rst

```diff
@@ -0,0 +1,194 @@
+*****************************
+Docs in Windows (Cygwin) 
+*****************************
+
+.. _cygwin-docs:
+
+Cygwin
+-------------------------------
+Cygwin is windows implementation of unix-like commands. You can basically use many linux commands from windows 
```

Comment:
- capitalize Windows
 - "*nix style commands"
 - avoid "basically" (it is meaningless)
 - capitalize Linux

> Cygwin is windows implementation of unix-like commands. You can basically use many linux commands from windows command prompt. (19 words in two sentences.)

> Cygwin is a Windows implementation of many GNU/Linux commands usable from the Windows command prompt. (16 words in one sentence.)

Try for brevity whenever possible.